src/drivers/input/i8042/i8042.c

```diff
@@ -0,0 +1,58 @@
+/*
+ * @file
+ *
+ * @date   10.04.2020
+ * @author Alexander Kalmuk
+ */
+
+#include <embox/unit.h>
+#include <drivers/i8042.h>
+
+EMBOX_UNIT_INIT(i8042_init);
+
+int i8042_read_mode(void) {
+	i8042_wait_write();
+	outb(I8042_CMD_READ_MODE, I8042_CMD_PORT);
+	i8042_wait_read();
+	return inb(I8042_DATA_PORT);
+}
+
+void i8042_write_mode(unsigned char val) {
+	i8042_wait_write();
+	outb(I8042_CMD_WRITE_MODE, I8042_CMD_PORT);
+	i8042_wait_write();
+	outb(val, I8042_DATA_PORT);
+}
+
+void i8042_write_aux(unsigned char val) {
+	i8042_wait_write();
+	outb(I8042_CMD_WRITE_MOUSE, I8042_CMD_PORT);
+	i8042_wait_write();
+	outb(val, I8042_DATA_PORT);
+}
+
+static int i8042_init(void) {
+	unsigned char mode;
```

Comment:
may be uint8_t?

Reply:
You're right, it's better for the driver. Fixed.